src/main/modules/nunjucks/index.ts

```diff
@@ -101,6 +103,10 @@ export class Nunjucks {
     nunjucksEnv.addGlobal('UnemploymentType', UnemploymentType)
     nunjucksEnv.addGlobal('BankAccountType', BankAccountType)
     nunjucksEnv.addGlobal('ClaimStatus', ClaimStatus)
+    nunjucksEnv.addGlobal('AppPaths', AppPaths)
+    if (FeatureToggles.isEnabled('finePrint')) {
+      nunjucksEnv.addGlobal('cookieText', 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.')
```

Comment:
@timja-kainos 
Is there a way that i could put a link to the cookies page in the cookie message. 

Similar to this
`
'We do this by using <a href="{{ cookieUrl }}">cookies</a>.',
          { cookieUrl: AppPaths.cookiesPage.uri}) | safe }}
`